components/org.wso2.carbon.identity.handler.event.account.lock/src/main/java/org/wso2/carbon/identity/handler/event/account/lock/AccountLockHandler.java

```diff
@@ -61,12 +61,12 @@ public String getName() {
     }
 
     public String getFriendlyName() {
-        return "Account Locking";
+        return "Account Lock";
     }
 
     @Override
     public String getCategory() {
```

Comment:
What's the usage of this getCategory() method?
Is this only used in UI?

Reply:
AFAIK, it used only in UI.